packages/amplify-graphql-generation-transformer/src/grapqhl-generation-transformer.ts

```diff
@@ -0,0 +1,219 @@
+import { GenerationDirective } from '@aws-amplify/graphql-directives';
```

Comment:
We're not because we can't. A customer must manually enable the model in their account. We hope to make some DX improvements here pre GA, but aren't there yet.